src/lua_console.cpp

```diff
@@ -0,0 +1,93 @@
+#include "lua_console.h"
+
+#include "catalua.h"
+#include "catacurse.h"
+
+#include "debug.h"
+#include "catacharset.h"
+
+lua_console::lua_console()
+{
+    cWin = newwin(lines, width, 0, 0);
+    iWin = newwin(1, width,lines, 0);
+}
+
+void lua_console::dispose()
+{
+    werase(cWin);
+    werase(iWin);
+    delwin(cWin);
+    delwin(iWin);
+}
+
+std::string lua_console::get_input()
+{
+    long key = 0;
+    int pos = -1;
+    return string_input_win(iWin, "", width, 0, 0, width, true, key, pos, "LUA", iWin->x, iWin->y, true, false,
```

Comment:
Similar problem here: the ncurses window doesn't have x and y members. You're supposed to call `getmaxx` and `getmaxy`.